spec/services/idv/analytics_events_enhancer_spec.rb

```diff
@@ -3,7 +3,15 @@
 RSpec.describe Idv::AnalyticsEventsEnhancer do
   let(:user) { build(:user) }
   let(:sp) { nil }
-  let(:session) { nil }
+  let(:user_session) { nil }
+  let(:session) do
+    if !user_session.nil?
+      {
+        'warden.user.user.session' => user_session,
+      }
+    end
```

Comment:
easier to understand positive checks IMO
```suggestion
    if user_session.present?
      {
        'warden.user.user.session' => user_session,
      }
    end
```